### PR TITLE
Added a new template to import huggingface models by taking access to…

### PR DESCRIPTION
**Pull request by rtdurga**
…ken and model id as parameters

## Checklist before requesting a review

- [x] I updated `CHANGELOG.MD` with a description of my changes
- [x] If the change was to a module, I ran the code validation script (`scripts/validate.sh`)
- [x] If the change was to a module, I have added thorough tests
- [x] If the change was to a module, I have added/updated the module's README.md
- [x] If a module was added, I added a reference to the module to the repository's README.md
- [x] I verified that my code deploys successfully using `seedfarmer apply`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kukushking** [comment]
Can you please update `README.MD` with the new template link in the description of SageMaker Project Templates module and `CHANGELOG.MD`?

File: modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/seed_code/build_app/ml_pipelines/training/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Training SageMaker Pipeline
+
+This SageMaker Pipeline definition creates a workflow that will:
+- Prepare the Abalone dataset through a SageMaker Processing Job
+- Train an XGBoost algorithm on the train set
+- Evaluate the performance of the trained XGBoost algorithm on the validation set
+- If the performance reaches a specified threshold, send the model for Manual Approval to SageMaker Model Registry.
```
Comment (LeonLuttenberger) [inline modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/seed_code/build_app/ml_pipelines/training/README.md:7]: This description isn't right. It applies to the `xgboost_abalone` template, but not this one.

Comment (rtdurga) [reply modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/seed_code/build_app/ml_pipelines/training/README.md]: Corrected the description. Please review

**Comment by kukushking** [comment]
Module checks are [failing](https://github.com/awslabs/aiops-modules/actions/runs/8904531434/job/24453945670?pr=71). Could you please run below to fix formatting?
```
scripts/fix.sh --language python --path modules/sagemaker/sagemaker-templates-service-catalog/
```

File: modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/product_stack.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        prod_account = aws_cdk.CfnParameter(
+            self,
+            "ProdAccountId",
+            type="String",
+            min_length=1,
+            max_length=16,
+            description="Prod account id.",
+        ).value_as_string
+
+        hf_access_token = aws_cdk.CfnParameter(
```
Comment (kukushking) [inline modules/sagemaker/sagemaker-templates-service-catalog/templates/hf_import_models/product_stack.py:83]: As discussed please change this to `hf_access_token_secret_name` and pull the secret from SSM in the pipeline runtime script. Thank you!

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: ab3129da4ba7ee40a3e0f824c052193793a6594f
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=367201f1-d7e3-4890-82c2-7f2711815838%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 6a8ad4a88d24b8a4b2be5ad71b7fc42113181515
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=35addf57-1227-4bc0-8ccc-132aa2ac44b3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: fc123280d00cf1c73168c288410aea5259924a0f
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=c9de60b9-f2de-4f4e-b38a-f1d6534dabd8%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 69a8bb25c4d370751f2879f57095b28f16987664
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7cf868b0-422f-4ea9-a4f4-909f48a9f414%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by kukushking — APPROVED** [review]
Looks great, thank you!

**Review by LeonLuttenberger — APPROVED** [review]

**Pull request merged**